app/adapters/application.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'DS' is not defined.<br>'from' is not defined.